test/cypress/integration/29-host.admin.test.js

```diff
@@ -156,17 +156,21 @@ describe('host dashboard', () => {
 
       // Mark as expired
       cy.getByDataCy('MARK_AS_EXPIRED-button').click();
-      cy.get('[data-cy="MARK_AS_EXPIRED-confirmation-modal"] [data-cy="confirmation-modal-continue"]').click();
+      cy.wait(500);
```

Comment:
Adding `cy.wait`s is usually a sign that we're doing something wrong; did you get any clues about the root cause?